docs/user-guide/contextual-assistants.rst

```diff
@@ -643,12 +369,21 @@ matter the context, you can use the :ref:`mapping-policy` to handle these. It wi
 predict the same action for an intent, and when combined with a forgetting mechanism,
 you don’t need to write any stories either.
 
+For example, let's say you see users having conversations like the following one with 
+your assistant, where they write a greeting in the middle of a conversation - 
+maybe because they were gone for a few minutes:
+
+.. image:: /_static/images/greet_interjection.png
+   :width: 240
+   :alt: Greeting Interjection
+   :align: center
```

Comment:
will add

Reply:
@akelad was going to add it, then realized Sara actually doesn't demonstrate the desired behaviour with this one since `greet` isn't handled with the mapping policy, and that's why I left it out =P

Reply:
aha! makes sense